docs/user/exams/students_guide.rst

```diff
@@ -0,0 +1,311 @@
+
+- The file browser displays the structure of the assignment. You can access any file within the assignment. The files content will then be displayed on the code editor where you can edit the file's content. 
+
+    - You can add new files and directories using the ``Add File``  and ``Add Folder`` buttons.
+    - You also have the ability to rename and delete files and folders, therefore **caution** is advised. 
+
+        .. note::
+            If you accidentally delete or remove a file, you can use ``Refresh files``, to load the last saved version from the server.
+
+- The code editor allows you to edit the content of specific files. It shows the line numbers and will also annotate the appropriate line, if a compilation error occurs. 
+- The instructions are docked to the right.
+- If you want to focus only on the code editor, you can collapse the instructions by pressing on the ``right arrow``. This can be reverted by pressing the arrow again. Similarly, if you want to collapse the file browser, you can press the ``left arrow`` above the file browser. 
+
+    .. note::
+        You can also choose to resize any of the three sections by dragging the ``outline box``.
+
+- When you press ``Save``, your files are saved on the Artemis server. However, you must press ``Submit`` for your solution to be counted!
+- When you press ``Submit``, your changes are pushed to the VC server and a build is started on the CI server. This is indicated by the results changing from ``No results found`` to ``Building and testing``.
+
+    .. warning::
+        There is no auto-submit!
+
+.. figure:: student/programming_exercises.png
+   :alt: Participating in Programming Exercises
+   :align: center
+
+   Participating in Programming Exercises with the online code editor and local IDE enabled
+
+- If your exercise allows the use of the local IDE you will have access to the button ``Clone Repository``.
+- When you click it you can choose to clone the exercise via ``HTTPS`` or ``SSH``, if you have configured your private key.
+
+    .. note::
+        You must link a public key to your account in advance if you want to use ``SSH``. 
+
+- To work offline follow these steps: 
+
+    1. Clone the Exercise
+    2. Import the project in your IDE
+    3. Work on the code
+    4. Commit and push the code. A push is equivalent to pressing the ``Submit`` button.
+
+.. figure:: student/clone_repository.png
+   :alt: Clone Repository
+   :align: center
+
+   Clone the Repository
+
+    .. warning::
+        You are responsible for pushing/submitting your code. Your instructors **cannot** help you if you have saved, but did not submit.
+
+- Real-time feedback in programming exercises during the online exam is limited.
+- You will only see if your code compiles or not
+
+    1. **0%, Build failed** means that your code does **not** compile!
+    2. **0%, 0 of 0 passed** means that your code compiles but provides no further information about your final score. 
```

Comment:
Isn't this only the case if all test cases are hidden?

Reply:
True, I rephrased this section: 

```
- Depending on your exam's configuration, your instructors limit the real-time feedback in programming exercises during the online exam.
- In that case, you will only see if your code compiles or not:

    1. **0%, Build failed** means that your code does **not** compile!
    2. **0%, 0 of 0 passed** means that your code compiles but provides no further information about your final score. 
```